src/main/java/zd/zero/waifu/motivator/plugin/WaifuMotivatorProject.java

```diff
@@ -87,6 +91,10 @@ private void checkIfInGoodState( Project projectOpened, Runnable onGoodState ) {
 
     @Override
     public void projectClosing( @NotNull Project project ) {
+        if(project == this.project) {
+            dispose();
```

Comment:
When a project was closed, the `disposed` was not being called for some reason, which would result in the idle listener emitting an event for a non-existent project.